src/parser/jobs/drk/modules/SaltAndDarkness.tsx

```diff
@@ -0,0 +1,80 @@
+import {t} from '@lingui/macro'
+import {Plural, Trans} from '@lingui/react'
+import {DataLink} from 'components/ui/DbLink'
+import {Event, Events} from 'event'
+import {Analyser} from 'parser/core/Analyser'
+import {filter} from 'parser/core/filter'
+import {dependency} from 'parser/core/Injectable'
+import {Data} from 'parser/core/modules/Data'
+import Suggestions, {SEVERITY, TieredSuggestion} from 'parser/core/modules/Suggestions'
+import React from 'react'
+import {isSuccessfulHit} from 'utilities'
+import {DISPLAY_ORDER} from './DISPLAY_ORDER'
+
+const SEVERITIES = {
+	UNUSED: {
+		1: SEVERITY.MEDIUM,
+		2: SEVERITY.MAJOR,
+	},
+	FAILED: {
+		1: SEVERITY.MEDIUM,
+		2: SEVERITY.MAJOR,
+	},
+}
+
+export class SaltAndDarkness extends Analyser {
+	static override handle = 'saltanddarkness'
+	static override title = t('drk.saltanddarkness.title')`Salt And Darkness`
+	static override displayOrder = DISPLAY_ORDER.RESOURCES
+
+	@dependency private data!: Data
+	@dependency private suggestions!: Suggestions
+
+	private saltedEarthUsed = 0
+	private saltAndDarknessUsed = 0
+	private saltAndDarknessDamaged = 0
+
+	override initialise() {
+		super.initialise()
+
+		const playerFilter = filter<Event>().source(this.parser.actor.id)
+		this.addEventHook(playerFilter.type('action').action(this.data.actions.SALTED_EARTH.id), () => this.saltedEarthUsed++)
+		this.addEventHook(playerFilter.type('action').action(this.data.actions.SALT_AND_DARKNESS.id), () => this.saltAndDarknessUsed++)
+		this.addEventHook(playerFilter.type('damage').cause(this.data.matchCauseAction(['SALT_AND_DARKNESS'])), this.onSaltDarknessDamage)
+		this.addEventHook('complete', this.onComplete)
+	}
+
+	private onSaltDarknessDamage(event: Events['damage']) {
+		if (isSuccessfulHit(event)) {
+			this.saltAndDarknessDamaged++
+		}
+	}
+
+	private onComplete() {
+		const unused = this.saltedEarthUsed - this.saltAndDarknessUsed
+		this.suggestions.add(new TieredSuggestion({
+			icon: this.data.actions.SALT_AND_DARKNESS.icon,
```

Comment:
I'd maybe swap the icon for one of these, just so it's easier to see when both are triggered.

Reply:
I don't really want to use the Salted Earth icon for "didn't use S&D", and it doesn't make sense to use it for "S&D didn't hit anything", so I'd rather keep the duplicated icon.